app/download/handler/download_era5.py

```diff
@@ -1,34 +1,24 @@
-from os import environ
+import datetime
+import tempfile
 
 import boto3
+from cdsapi.api import Client, Result  # type: ignore
 
-from .download import download_data
-
-
-class CDSAPINotAvailableYet(RuntimeError):
-    pass
-
-
-class CDSAPITooManyRequests(RuntimeError):
-    pass
+BUCKET = "odin-era5"
 
 
 def lambda_handler(event, context):
-    ssm = boto3.client("ssm", region_name="eu-north-1")
-
-    # Get the parameter
-    key = ssm.get_parameter(Name="/odin/cdsapi/key", WithDecryption=True)
-    url = ssm.get_parameter(
-        Name="/odin/cdsapi/url",
-    )
-    environ["CDSAPI_KEY"] = key["Parameter"]["Value"]
-    environ["CDSAPI_URL"] = url["Parameter"]["Value"]
-    try:
-        result = download_data(event["date"], "pl", event["hour"])
-        return result
-    except Exception as err:
-        if "too many" in str(err):
-            raise CDSAPITooManyRequests()
-        if "yet" in str(err):
-            raise CDSAPINotAvailableYet()
-        raise err
+    s3_client = boto3.client("s3")
+    client = Client(progress=False, wait_until_complete=False)
+    result = Result(client, event["reply"])
+    with tempfile.NamedTemporaryFile() as f:
```

Comment:
I don't see that this Lambda has `ephemeral_storage` activated; does this work without it?

Reply:
512Mb is always available in /tmp